jekyll/_cci2/adv-config.md

```diff
@@ -58,6 +126,57 @@ For more information on using shell scripts in your config, see the [Using Shell
 
 Use Selenium to manage in-browser testing:
 
+{:.tab.executors_three.Cloud}
+```yaml
+version: 2.1
+
+orbs: 
+  browser-tools: circleci/browser-tools@1.2.3
+jobs:
+  build:
+    docker:
+      - image: cimg/node-browsers
+        auth:
+          username: mydockerhub-user
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+      - checkout
+      - run: mkdir test-reports
+      - run:
+          name: Download Selenium
+          command: curl -O http://selenium-release.storage.googleapis.com/3.5/selenium-server-standalone-3.5.3.jar
+      - run:
+          name: Start Selenium
+          command: java -jar selenium-server-standalone-3.5.3.jar -log test-reports/selenium.log
+          background: true
+```
+
+{:.tab.executors_three.Server_3}
+```yaml
+version: 2.1
+
+orbs: 
+  browser-tools: circleci/browser-tools@1.2.3
+jobs:
+  build:
+    docker:
+      - image: cimg/node-browsers
+        auth:
+          username: mydockerhub-user
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+      - checkout
+      - run: mkdir test-reports
```

Comment:
If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`